metadata/stream/rationale.yaml

```diff
@@ -141,6 +159,8 @@ architectures:
         # string, and represents advice rather than a value we might
         # change.
         image: Fedora:CoreOS:stable:latest
+      azurestack:
+        image: Fedora:CoreOS:stable:latest
```

Comment:
These have platform-specific syntax, and should only be included in cases where the platform has some global namespace in which FCOS images are available.  (Either because we upload them, or because the cloud automatically fetches them from us.)  I don't think that applies to any of the platforms added here.  (Exoscale provides FCOS images but they're out of date; see https://github.com/coreos/fedora-coreos-tracker/issues/1166.)

Reply:
Actually, Exoscale does have a central registry, so let's include it as aspirational functionality we might support in the future.  It [looks like](https://github.com/coreos/fedora-coreos-docs/pull/391/files#diff-903c74bb36d203fe0c8e60c9ebbb3ecdf58c892ba88551c83eb886e34fde3c4bL35) we'd want something like:

```suggestion
        image: Linux Fedora CoreOS 64-bit
```